[DRAFT] Portuguese translation for the learn folder (#1)
eosdetroit/wax-developer: docs/pt/learn/about-wax/what-is-wax.md

---
title: O que é WAX?
order: 1
---

# O que é a WAX blockchain?

> A WAX Blockchain, reconhecida como rei dos NFTs, é uma plataforma líder de blockchain de camada 1 que revoluciona a criação, o comércio e a propriedade de ativos digitais. Ela possui a maior base de usuários e volume de transações do setor, conforme verificado pelo [Dappradar.com] (https://dappradar.com/). A missão da WAX é tornar os NFTs acessíveis ao mercado de massa por meio de um mercado seguro, ecológico e fácil de usar.

Reviewer: ```suggestion
A Blockchain WAX, reconhecida como Soberana dos NFTs, é uma plataforma líder de blockchain de camada 1 que revoluciona a criação, o comércio e a propriedade de ativos digitais. Ela possui a maior base de usuários e volume de transações do setor, conforme verificado pelo [Dappradar.com] (https://dappradar.com/). A missão da WAX é tornar os NFTs acessíveis ao mercado de massa por meio de um mercado seguro, ecológico e fácil de usar.
```

Fundada em 2017 por William E. Quigley e Jonathan Yantis, a WAX facilitou a troca de mais de 360 milhões de colecionáveis digitais, colaborando com marcas renomadas como NASCAR, AMC, Mattel e Hasbro. A WAX foi pioneira na conexão de NFTs a produtos físicos de consumo, oferecendo possibilidades interessantes para empresas de produtos de consumo, conhecidas como vIRLs®.

> A WAX estende sua influência para além dos colecionáveis, servindo como uma rede líder para NFTs de entretenimento e jogos web3 no metaverso. Jogos de blockchain notáveis, como Alien Worlds e Splinterlands, encontram seu lar na plataforma WAX.  Seu sucesso está em sua infraestrutura de fácil utilização,permitindo experiências perfeitas para entusiastas de criptoativos e novatos, sem taxas de transação e com escalabilidade impressionante.

Reviewer: ```suggestion
A WAX estende sua influência para além dos colecionáveis, servindo como uma rede líder para NFTs de entretenimento e jogos web3 no metaverso. Jogos de blockchain notáveis, como Alien Worlds e Splinterlands, encontram seu lar na plataforma WAX. Seu sucesso está em sua infraestrutura de fácil utilização, permitindo experiências perfeitas para entusiastas de criptoativos e novatos, sem taxas de transação e com escalabilidade impressionante.
```

> Operando em um mecanismo de consenso DPoS, a WAX garante transações rápidas e alto rendimento, atendendo às demandas do comércio de ativos digitais. Além disso, seus recursos de interoperabilidade permitem a integração com outras blockchains e sistemas externos, aumentando o alcance e a utilidade dos ativos digitais criados pelo WAX.

Reviewer: ```suggestion
Operando em um mecanismo de consenso DPoS, a WAX garante transações rápidas e alto rendimento, atendendo às demandas do comércio de ativos digitais. Além disso, seus recursos de interoperabilidade permitem a integração com outras blockchains e sistemas externos, aumentando o alcance e a utilidade dos ativos digitais criados pela WAX.
```

> A WAX Blockchain tem uma comunidade profunda e incrível, repleta de artistas, criadores, desenvolvedores e empreendedores que se unem para aprimorar as ferramentas e os recursos da blockchain. Alguns desenvolvedores forneceram ferramentas fáceis para criar e lançar vendas de comércio digital e outros até mesmo se aprofundaram no espaço digital para físico. Equipes como [BountyBlok] (https://www.bountyblok.io/), [NFT Hive] (https://nfthive.io/) e [Atomic Hub] (https://wax.atomichub.io/) são apenas alguns exemplos de como essa comunidade facilitou a criação, a compra, a venda e o comércio na WAX Blockchain.

Reviewer: ```suggestion
A Blockchain WAX tem uma comunidade intensa e incrível, repleta de artistas, criadores, desenvolvedores e empreendedores que se unem para aprimorar as ferramentas e os recursos da blockchain. Alguns desenvolvedores forneceram ferramentas fáceis para criar e lançar vendas de comércio digital e outros até mesmo se aventuraram no espaço do digital para o físico. Equipes como [BountyBlok] (https://www.bountyblok.io/), [NFT Hive] (https://nfthive.io/) e [Atomic Hub] (https://wax.atomichub.io/) são apenas alguns exemplos de como essa comunidade facilitou a criação, compra, venda e comércio na Blockchain WAX.
```

> Como uma potência descentralizada, a WAX impulsiona a adoção de ativos digitais e NFTs, criando oportunidades para criadores, colecionadores e empresas. Para saber mais sobre a WAX, visite [https://wax.io](https://wax.io) e mantenha-se atualizado por meio de seu [Twitter](https://twitter.com/WAX_io) e [canais Discord](https://discord.com/invite/dJtPetMdfb).

Reviewer: ```suggestion
Como uma potência descentralizada, a WAX impulsiona a adoção de ativos digitais e NFTs, criando oportunidades para criadores, colecionadores e empresas. Para saber mais sobre a WAX, visite [https://wax.io](https://wax.io) e mantenha-se atualizado por meio de seu [Twitter](https://twitter.com/WAX_io) e [canais do Discord](https://discord.com/invite/dJtPetMdfb).
```
